newIDE/app/src/UI/Theme/index.js

```diff
@@ -1,23 +1,61 @@
 // @flow
+import {
+  select,
+  blend as Install,
+} from '../../Utils/Object';
+import { isLTR } from '../../Utils/i18n/language';
+import domain from './domain';
+import DefaultTheme, {
+  themeName as defaultThemeName,
+  type Theme,
+} from './DefaultTheme';
 import DarkTheme from './DarkTheme';
-import DefaultTheme from './DefaultTheme';
 import NordTheme from './NordTheme';
 import SolarizedDarkTheme from './SolarizedDarkTheme';
-import { type Theme } from './DefaultTheme';
 import './Global.css';
 
+type MuiTheme = $Exact<typeof DefaultTheme.muiTheme>;
+
+/**
+ * Enhances every Theme instance with a `muiTheme` property accessor.
+ *
+ * This ensures the `muiTheme` is correctly applied whenever language
+ * directionality is changed. Thus, based on `language-ltr`, one of
+ * two _distinct_ `muiTheme` instances is returned; these two instances
+ * are exactly alike (i.e. have the exact same properties).
+ */
+const install = new Install(({ gdevelopTheme, muiTheme }: Theme): Theme => {
```

Comment:
Let's remove this Install/`blend` and make instead a check for the language rtl value in `getTheme`, and use a function dedicated to generate a rtl version of the muiTheme.